readme.md

```diff
@@ -50,6 +52,18 @@ db.connect({
 });
 ```
 
+#### Retry
+
+Retry configuration can be passed well during the initialisation. The retry mechanism is based on [p-retry](https://github.com/sindresorhus/p-retry) and requires the same [options](https://github.com/tim-kos/node-retry#retryoperationoptions). Configuring the retry will allow the user to automatically retry the dynongo operation if it's an retryable error.
+
+```js
+db.connect({
+	retryOptions: {
```

Comment:
Maybe we should just go with `retry` here?

Reply:
Ok, will change it to a number as well and not the options from `p-retry`